anvil-python/anvil/utils.py

```diff
@@ -43,3 +60,84 @@ def __call__(self, *args, **kwargs):  # type: ignore[no-untyped-def]
             LOG.warn(message)
             LOG.error("Error: %s", e)
             sys.exit(1)
+
+
+# We reimplement from sunbeam to avoid openstack dependencies
+class UpgradeCharm(BaseStep, JujuStepHelper):
```

Comment:
It's fine that it was re-implemented. Could you please move it under `upgrades/inter_channel` like the relevant functions of sunbeam?

Reply:
Plus there is no need for the comment :slightly_smiling_face: